lib/charms/postgresql_k8s/v0/postgresql_tls.py

```diff
@@ -139,7 +141,10 @@ def _on_certificate_available(self, event: CertificateAvailableEvent) -> None:
         self.charm.set_secret(SCOPE, "ca", event.ca)
 
         try:
-            self.charm.push_tls_files_to_workload()
+            if not self.charm.push_tls_files_to_workload():
```

Comment:
We'll have to update the other charms (PG VM and PGB K8S) or check explicitly for false, so that None will not trigger a defer.

Reply:
Exactly. I'm preparing PRs for the other PG and PgBouncer repos.

Reply:
Are we creating a strict binding: PG rev 109+ requires PGB revision 58+?
Should we start building compatibility matrix inside charm.py to check before deployment and warn users on incompatibilities... or create backward compatible code.

Reply:
No Alex, we aren't creating that strict binding. What we need to do is update PgBouncer code in a similar way as here (adding the returns) when we upgrade the TLS lib there (in [this PR](https://github.com/canonical/pgbouncer-k8s-operator/pull/111)).